package.json

```diff
@@ -117,7 +117,7 @@
     "codelyzer": "^5.0.1",
     "core-js": "^3.1.4",
     "cross-env": "^5.2.0",
-    "dtslint": "^0.8.0",
+    "raynos-dtslint": "1.0.4",
```

Comment:
why raynos-dtslint?

Reply:
Because we are forced to use TS 3.4.5 in order to build with the angular compiler and dtslint brings in Typescript@next which breaks the dtslint build.
The `raynos-dtslint` package is a fork locked to TS 3.4.5 which helps us solve our issue.
See this for the reason for this forked package:
https://github.com/microsoft/dtslint/issues/231
https://github.com/microsoft/dtslint/pull/234

We will move back to the main `dtslint` again as soon as we can.

Reply:
Ok